src/test/scala/nl.knaw.dans.easy.download/ServletSpec.scala

```diff
@@ -55,15 +55,15 @@ class ServletSpec extends TestSupportFixture with EmbeddedJettyContainer
   addServlet(new EasyDownloadServlet(app), "/*")
 
   private def expectDownloadStream(path: Path) = {
-    (app.http.copyHttpStream(_: URI)) expects new URI(s"http://localhost:20110/bags/$uuid/${ path.escapePath }") once()
+    (app.http.copyHttpStream(_: URI)) expects new URI(s"http://localhost:20110/bags/$uuid/${ path.escapePath }") noMoreThanOnce()
```

Comment:
`noMoreThanOnce` means either 0 or 1 times, right? I'd rather have that explicit.

Suggestion for a solution: provide an extra parameter (integer) to these functions (this one and below) and use [`repeat`](https://www.javadoc.io/doc/org.scalamock/scalamock_2.12/latest/org/scalamock/handlers/CallHandler.html#repeat(count:Int):org.scalamock.handlers.CallHandler[R]) instead of `noMoreThanOnce()`

Reply:
done